include/PR/hardware.h

```diff
@@ -1,6 +1,8 @@
 #ifndef _ULTRA64_HARDWARE_H_
 #define _ULTRA64_HARDWARE_H_
```

Comment:
Just want to note this didn't get it header guard changed. However, I think that is actually for the best since this isn't a real libultra header, and will be removed soon(tm) anyways.

Reply:
Yeah, that's exactly why I didn't bother cleaning it up